<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSelectZLayers</class>
 <widget class="QDialog" name="DSelectZLayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Z-layers</string>
  </property>
  <widget class="QLabel" name="textLabel7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This dialog allows you to select the Z-layers which will be considered for plotting the dislocation displacement map. You can select at least one and at most all Z-layers read in from the plot file. Arrows between the atoms in the Z-layers that are NOT selected here will not be plotted.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>176</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>351</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Z-layers in the plot file</string>
   </property>
   <widget class="QListWidget" name="zLayers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>231</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>list of all Z-layers in the current plot</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btSelAll">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>selects all Z-layers in the block</string>
    </property>
    <property name="text">
     <string>&amp;Select all Z-layers</string>
    </property>
    <property name="shortcut">
     <string>Alt+S</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btUnselAll">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>unselects all Z-layers in the block</string>
    </property>
    <property name="text">
     <string>&amp;Unselect all Z-layers</string>
    </property>
    <property name="shortcut">
     <string>Alt+U</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="applyHow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Apply this settings to the current plot only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Apply this settings to all selected plots</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btSelAll</tabstop>
  <tabstop>applyHow</tabstop>
  <tabstop>btUnselAll</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DSelectZLayers</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DSelectZLayers</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
